src/app/modules/spcp/spcp.util.ts

```diff
@@ -65,11 +65,7 @@ export const isValidAuthenticationQuery = (
   destination: string,
   idpPartnerEntityId: string,
 ): boolean => {
-  return (
-    !!destination &&
-    isArtifactValid(idpPartnerEntityId, samlArt) &&
-    DESTINATION_REGEX.test(destination)
-  )
+  return !!destination && isArtifactValid(idpPartnerEntityId, samlArt)
```

Comment:
should we update the regex instead of completely removing it? Or is the regex unneeded after all?